tutorials/prompts/react_prompting.ipynb

```diff
@@ -50,7 +50,7 @@
    "metadata": {},
```

Comment:
<p>Is the definition of mostly_correct in the prompt? if not I would just put the prompt here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/9291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>